app.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name": "Shynet",
+  "description":"Modern, privacy-friendly, and detailed web analytics that works without cookies or JS.",
+    "keywords":[
+      "app.json",
+      "shynet",
+      "heroku",
+      "analytics",
+      "privacy",
+      "friendly"
+  ],
+  "website": "https://github.com/milesmcc/shynet",
+  "repository": "https://github.com/milesmcc/shynet",
+  "logo": "https://github.com/milesmcc/shynet/raw/master/images/slogo.png",
+  "success_url": "/",
+  "stack": "container",
+  "addons": [
+    "heroku-postgresql:hobby-dev"
+  ],
+  "formation": {
+    "web": {
+      "quantity": 1,
+      "size": "free"
+    }
+  },
+  "env": {
+    "DB_NAME": {
+      "description": "Database name (PostgreSQL)",
+      "value": "shynet",
+      "required": false
+    },
+    "DB_USER": {
+      "description": "Database username",
+      "value": "",
+      "required": false
+    },
+    "DB_PASSWORD": {
+      "description": "Database password",
+      "value": "",
+      "required": false
+    },
+    "DB_HOST": {
+      "description": "Database hostname",
+      "value": "",
+      "required": false
+    },
+    "DB_PORT": {
+      "description": "Database port",
+      "value": "5432",
+      "required": false
+    },
```

Comment:
I think I might be missing something here. If we're adding support for `DATABASE_URL`, why do we need to provide these? Can we rely on Heroku's automatically set `DATABASE_URL`?

Reply:
yes, I think we can.  
Because even in other programming languages "heroku's stack" I receve `DATABASE_URL`. 

It is set when you add cleardb or heroku-postgresql addons. Either one will add the same variable.

Reply:
Alright, got it. Can we then specify that these are explicitly not required if you're using the builtin Heroku Postgres addon?

Reply:
By removing `{DB_NAME, DB_USER, DB_PASSWORD, DB_HOST}` from `app.json` we wont be able to use with a different database then "addon heroku" itselfs.

Heroku only sets `DATABASE_URL` when you **add the addon** it with `heroku addons:create heroku-postgresql:hobby-dev`, for that reason I kept in on ` app.json`.